backend/src/main/java/OpenEndedQuestion.java

```diff
@@ -0,0 +1,41 @@
+import javax.persistence.Entity;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/*
+ * A class to store open-ended questions
+ */
+@Entity
+public class OpenEndedQuestion extends Question {
+
+    private List<String> answers;
+
+
+    public OpenEndedQuestion() {
+        super();
+        this.answers = new ArrayList<>();
+    }
+
+
+    public OpenEndedQuestion(String question) {
+        super(question);
+        this.answers = new ArrayList<>();
+    }
+
+
+    public List<String> getAnswers() {
+        return answers;
+    }
+
+
+    public void setAnswers(ArrayList<String> answers) {
+        this.answers = answers;
+    }
+
+
+    public void addAnswer(String answer) {
```

Comment:
return the result of the add operation

Reply:
based on what condition would it return true or returns false?

Reply:
it return what List.add returns, literally just add a return to the operation